lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'dish'
+
+class Order
+
+	attr_reader :current_order
```

Comment:
It doesn't make sense for and `order` object to have a `current_order` attribute.